### ISSUE TITLE
Seth and Vaulter Battle causes game freeze

### ISSUE THREAD

**Issue by SdeSpencer** [lushen124/Universal-FE-Randomizer#20]
Not my problem (noticed in a fairly recent stream, using what i believe was Yune 0.7.3), but the screen froze at Vaulter and Seth's battle during the prologue. Seth was a Sage and Vaulter was a pegasus knight. [The Stream at hand](https://www.twitch.tv/videos/371070690) (freeze is a bit after 14 minutes in) 

I don't know if this is something that can't be fixed or isn't worth fixing (i wouldn't be surprised if it is) but it was mentioned somewhere that this battle is part of the reason one or both are locked to certain classes, so figure that it would be worth reporting

**Comment by lushen124** [comment]
It's definitely worth fixing! 

Yeah, I ran into this as well. I'm not sure of your exact randomization results, but I would guess some palette shenanigans that I screwed up because I took a very naive approach initially to palettes, especially for not-usually playable classes and classes that don't have a reference palette to work with, outside of the generic one baked into the sprite. I've resolved most palette issues in the upcoming version. Hopefully I can get that out soon, because there are a LOT of fixes in there.
